examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -7154,6 +7154,301 @@
             }
           ]
         },
+        {
```

Comment:
shouldn't there be generated files to go with this change?

Reply:
Generated files will be added to the PR once it is otherwise approved. It simply adds too much noise to the file list/diffs